azure_functions_worker/dispatcher.py

```diff
@@ -299,12 +342,11 @@ def update_opentelemetry_status(self):
 
             self._context_api = context_api
             self._trace_context_propagator = TraceContextTextMapPropagator()
-            self._otel_libs_available = True
 
             logger.info("Successfully loaded OpenTelemetry modules. "
                         "OpenTelemetry is now enabled.")
         except ImportError:
-            self._otel_libs_available = False
+            self._azure_monitor_available = False
```

Comment:
Do we still need to set `_azure_monitor_available` to false here? It has a default value of false, and it will be set to false in `initialize_azure_monitor` if there's an ImportError

Reply:
Changed to log an exception instead.